osf/models/node.py

```diff
@@ -393,7 +393,7 @@ def is_preprint_orphan(self):
     def preprint_url(self):
         if self.is_preprint:
             try:
-                return self.preprint.url
+                return self.preprints.first().url
             except IndexError:
```

Comment:
I *think* this needs to be changed to `AttributeError` because IIRC, `.first()` can return `None`.